src/pipecat/services/ai_services.py

```diff
@@ -169,6 +169,7 @@ def __init__(
         self._push_stop_frames: bool = push_stop_frames
         self._stop_frame_timeout_s: float = stop_frame_timeout_s
         self._sample_rate: int = sample_rate
+        self._settings: Dict[str, Any] = {}
```

Comment:
is there a reason why this and the `_update_settings()` function can't be in `AIService` and then you wouldn't need to have it in three places. The only difference I see is the logging but i think that would be fine to be more generic.

Reply:
Moved it to the base AIServices class. Allowing the TTSService and STTService classes to override as they need special handling for `language`.